.gitignore

```diff
@@ -1,5 +1,5 @@
 .#*
-#*#
+\#*#
```

Comment:
Just a plain `#` starts a comment, so this wasn't catching files like `#emr.py#` created by emacs.